Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -185,6 +185,15 @@ class InitialValues {
     static CSS::TransformBox transform_box() { return CSS::TransformBox::ViewBox; }
     static CSS::Direction direction() { return CSS::Direction::Ltr; }
     static CSS::UnicodeBidi unicode_bidi() { return CSS::UnicodeBidi::Normal; }
+    static CSS::Transformation rotate()
+    {
+        Vector<TransformValue> values;
+        values.append({ Number(Number::Type::Number, 0) });
+        values.append({ Number(Number::Type::Number, 0) });
+        values.append({ Number(Number::Type::Number, 0) });
+        values.append({ Angle::make_degrees(0) });
+        return CSS::Transformation(CSS::TransformFunction::Rotate3d, move(values));
+    }
```

Comment:
This doesn't seem like a valid transformation to me? The initial value is `none`, so we can instead make `ComputedValues::m_noninherited.rotate` be an `Optional<CSS::Transform>` that defaults to empty. Then when we want the current transformation matrix, we can just skip the rotation step if it's empty.